app/main/gtm.py

```diff
@@ -32,7 +32,7 @@ def detect_gtm_anon_id():
     @after_this_request
     def remember_gtm_anon_id(response):
         session["gtm_anon_id"] = str(uuid.uuid4())
-        expiration_date = datetime.now(UTC) + timedelta(days=30)
+        expiration_date = datetime.utcnow() + timedelta(days=30)
```

Comment:
This function has been depreciated in favour of now(UTC)
[https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow)

Reply:
Updated to use `datetime.now(timezone.utc)`